mmv1/templates/terraform/examples/dataproc_metastore_federation_bigquery.tf.erb

```diff
@@ -18,7 +17,6 @@ resource "google_dataproc_metastore_federation" "<%= ctx[:primary_resource_id] %
 }
 
 resource "google_dataproc_metastore_service" "<%= ctx[:primary_resource_id] %>" {
-  provider   = google-beta
```

Comment:
Can you also remove google-beta provider in line 32 of this file?

Reply:
good catch, thanks